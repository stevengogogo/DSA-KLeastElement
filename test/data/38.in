10 50000
-4244 -9811 1171 -3176 4116 -6314 6506 -2002 -7604 2489
Delete 9
Query 3 6 3
Insert 9 -6442
Reverse 2 2
Reverse 5 5
Query 10 10 1
Query 3 10 8
Query 9 9 1
Reverse 8 8
Query 10 10 1
Query 5 10 6
Query 8 8 1
Insert 8 1836
Reverse 9 11
Delete 11
Query 9 9 1
Query 5 5 1
Reverse 2 2
Query 1 2 2
Reverse 10 10
Query 8 9 2
Reverse 5 8
Insert 10 -7669
Delete 7
Reverse 3 5
Insert 4 8333
Query 2 2 1
Reverse 7 7
Delete 5
Query 2 8 1
Reverse 6 7
Reverse 2 10
Query 7 9 1
Reverse 5 5
Query 9 9 1
Reverse 7 10
Query 8 8 1
Query 5 7 2
Delete 8
Delete 6
Delete 3
Delete 5
Query 3 3 1
Query 6 6 1
Insert 2 8437
Delete 7
Delete 2
Reverse 2 3
Reverse 5 5
Reverse 1 3
Delete 2
Reverse 1 1
Insert 2 8700
Reverse 1 1
Reverse 2 2
Delete 2
Insert 1 7262
Reverse 4 4
Query 4 4 1
Delete 4
Query 1 4 4
Delete 1
Query 3 3 1
Delete 1
Insert 3 -2402
Query 2 3 1
Insert 4 7818
Reverse 2 2
Query 4 4 1
Reverse 3 3
Delete 2
Query 1 3 1
Insert 3 9237
Delete 2
Query 3 3 1
Query 3 3 1
Insert 4 -2779
Delete 2
Insert 4 1048
Delete 2
Query 1 2 2
Insert 4 1397
Delete 1
Delete 1
Reverse 2 2
Reverse 1 1
Query 2 2 1
Insert 3 -1430
Delete 2
Reverse 2 2
Reverse 2 2
Insert 3 6531
Query 1 3 2
Insert 2 -6141
Insert 4 5835
Query 4 4 1
Query 4 4 1
Reverse 4 4
Insert 2 -4580
Query 5 6 2
Reverse 5 5
Query 3 3 1
Insert 3 -4135
Reverse 7 7
Reverse 1 6
Query 5 5 1
Delete 1
Insert 5 1570
Query 4 5 2
Delete 5
Query 6 6 1
Delete 4
Query 3 5 1
Query 5 5 1
Delete 2
Query 1 3 2
Reverse 1 3
Insert 1 -2085
Delete 5
Reverse 2 3
Delete 1
Reverse 3 3
Reverse 1 3
Query 3 3 1
Delete 1
Query 1 1 1
Query 2 2 1
Query 1 1 1
Insert 3 -4017
Insert 2 -3001
Query 2 4 3
Reverse 2 4
Query 3 3 1
Query 2 3 1
Insert 4 -877
Reverse 4 4
Delete 4
Reverse 1 2
Query 2 2 1
Delete 3
Delete 3
Query 2 2 1
Insert 2 7975
Reverse 3 3
Query 1 2 1
Reverse 2 2
Insert 4 4397
Delete 3
Reverse 2 3
Reverse 1 1
Delete 2
Query 2 2 1
Delete 1
Reverse 1 1
Query 1 1 1
Query 1 1 1
Insert 1 7092
Reverse 1 2
Reverse 1 1
Delete 1
Delete 1
Insert 1 4395
Insert 1 109
Query 1 1 1
Query 2 2 1
Insert 2 -6144
Delete 3
Reverse 2 2
Delete 2
Reverse 1 1
Query 1 1 1
Reverse 1 1
Reverse 1 1
Query 1 1 1
Query 1 1 1
Reverse 1 1
Delete 1
