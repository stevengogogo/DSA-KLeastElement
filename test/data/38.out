1171
2489
6506
-6442
2489
6506
-2002
2489
4116
-4244
2489
-9811
-9811
1171
1836
1836
4116
2489
1171
6506
8333
8333
-2402
7818
-4244
7818
7818
-2779
1397
1048
5835
5835
6531
-6141
-4580
1570
6531
-4135
6531
-1430
-4135
1048
-4135
1048
-3001
-4135
-4135
1048
1048
-4017
7975
7975
7975
109
4395
109
109
109
